cnxdb/tests/cli/test_main.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+
+import psycopg2
+import pytest
+
+from .. import testing
+
+
+def _translate_parts_to_args(parts):
+    """Translates connection string parts to arguments"""
+    return ['-h', parts['host'],
+            '-p', parts['port'],
+            '-d', parts['dbname'],
+            '-U', parts['user'],
+            ]
+
+
+def test_init(connection_string_parts, db_wipe):
```

Comment:
Come to think about it, I think it would be even nicer if you write a `db` fixture, rather than the `db_init` and `db_wipe`, so the `db.init()` can be called if necessary, and `db` is going to clean itself up.